### ISSUE TITLE
Path is ignored from base

### ISSUE THREAD

**Issue by basilfx** [danielgtaylor/restish#158]
I believe this is related to #5 (more specific, [this](https://github.com/danielgtaylor/restish/issues/5#issuecomment-716189877) comment).

I have an API in my `apis.json`:

```json
{
  "backend": {
    "base": "https://some-api.azurewebsites.net/api",
    "spec_files": ["/Users/basilfx/.restish/openapi/openapi.some-api.v1.json"],
    "profiles": {
        ...
    }
  }
}
```

The OpenAPI 3.0 spec contains an endpoint, lets say `/some-route/get-all`

The base contains a path (prefix), but it ignored when invoked:

```
$ restish backend -v get-all-some-resource
GET /some-resource/get-all HTTP/1.1
...
```

That is wrong, I would expect:

```
$ restish backend -v get-all-some-resource
GET /api/some-resource/get-all HTTP/1.1
...
```

I tried several things:

* Remove the trailing slash from `apis.json`.
* Delete the cache files.
* Remove the `servers` part from the OpenAPI spec.
* Synchronizing.

It does work if prefix the operation in the OpenAPI spec with `/api`.

Version: Restish 0.16.0 on macOS 13.1.

**Comment by wmendes-ionos** [comment]
@basilfx could you maybe checkout this https://github.com/danielgtaylor/restish/issues/158 and tell us if it fixes your issue ?

**Comment by danielgtaylor** [comment]
This should be fixed in #185.
